scipy/integrate/_quadpack_py.py

```diff
@@ -435,6 +435,10 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     if not isinstance(args, tuple):
         args = (args,)
 
+    # if a == b, directly return 0 with error 0
+    if a == b:
+        return [0, 0]
```

Comment:
Do we want to handle the case where `full_output` is non-zero? Users would be expecting an `infodict` in the return tuple as well in this case.

Reply:
As a matter of fact, yes. Looks like this is why the refguide checks are failing. I will submit a PR targeting your branch, @Shengyis, that hopefully will address this.